shared/components/ConferenceHightlights/index.js

```diff
@@ -27,50 +27,46 @@ const data = {
     // },
   ],
   youtube: [
-    // {
-    //   href: 'https://www.youtube.com/',
-    //   src: 'https://res.cloudinary.com/red-badger-assets/image/upload/v1486565031/react-london-2017/jani_evakallio_photo.jpg',
-    //   name: 'Jani Eväkallio',
-    //   company: 'Formidable Labs',
-    //   title: 'Lightning talk: Offline for the greater good',
-    // },
-    // {
-    //   href: 'https://www.youtube.com/',
-    //   src: 'https://pbs.twimg.com/profile_images/829282867020722177/el35E312.jpg',
-    //   name: 'Anna Doubkova',
-    //   company: 'Red Badger',
-    //   title: 'Lightning talk: Offline for the greater good',
-    // },
-    // {
-    //   href: 'https://www.youtube.com/',
-    //   src: 'https://res.cloudinary.com/red-badger-assets/image/upload/v1486054146/react-london-2017/Ken_Wheeler_small.jpg.jpg', // eslint-disable-line
-    //   name: 'Ken Wheeler',
-    //   company: 'Formidable Labs',
-    //   title: 'Lightning talk: Offline for the greater good',
-    // },
+    {
+      href: 'https://www.youtube.com/watch?v=ziAShzxAVKY&t=12s',
```

Comment:
I agree, it looks like a mistake from when the link was originally copied and pasted. Other than that though, this is looking good